src/Workspaces/Core/Portable/Workspace/Solution/ProjectDependencyGraph.cs

```diff
@@ -21,7 +25,7 @@ public class ProjectDependencyGraph
         private readonly NonReentrantLock _dataLock = new NonReentrantLock();
 
         // These are computed fully on demand. null or ImmutableArray.IsDefault indicates the item needs to be realized
-        private ImmutableDictionary<ProjectId, ImmutableHashSet<ProjectId>> _lazyReverseReferencesMap;
+        private ImmutableDictionary<ProjectId, ImmutableHashSet<ProjectId>>? _lazyReverseReferencesMap;
```

Comment:
Why can this be null now versus empty? What's the difference of those two values?

Reply:
I didn't change this. `_lazyReverseReferencesMap` is lazily- but fully-computed. `null` means it hasn't been computed, and any other value means it has been fully computed for all projects.

Reply:
➡️ Currently not planning to change this

Reply:
Ah, I got thrown off by the comment. Funky we have a comment about two fields that have different semantics, but whatever.